README.md

```diff
@@ -9,3 +9,8 @@ Learning how to use scss styles in html pages.
 When we use [Sass](https://sass-lang.com/), we prefer Scss syntax. For Css,
 we convert Scss to Css with Sass. We prefer to install `npm` instead of `brew`
 or `choco` to have a simple single run file. We use `npx sass` for convenience.
+
+### Using Variables
+
+We use variables to make it easier to apply the theme's colors and fonts and
+to improve readability.

```

Comment:
"We use variables so that applying theme and reading code becomes more convenient" diyebiliriz bence